ws-nextjs-app/pages/[service]/send/[id]/SubmitButton/index.test.tsx

```diff
@@ -19,28 +18,3 @@ describe('SubmitButton', () => {
     expect(button).toMatchSnapshot();
   });
 });
-
-describe('handleSubmit', () => {
```

Comment:
I've removed the button 'submit' tests for now as the submit handling will change quite a bit from the initial implementation. Thought its best for now so that it doesn't hold up spike development.